sdk/storage/Azure.ResourceManager.Storage/samples/Generated/Samples/Sample_StorageAccountResource.cs

```diff
@@ -815,7 +815,7 @@ public async Task RegenerateKey_StorageAccountRegenerateKey()
         public async Task GetAccountSas_StorageAccountListAccountSAS()
         {
             // Generated from example definition: specification/storage/resource-manager/Microsoft.Storage/stable/2023-05-01/examples/StorageAccountListAccountSAS.json
-            // this example is just showing the usage of "StorageAccounts_ListAccountSas" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "StorageAccounts_ListAccountSAS" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `SAS`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/storage/resource-manager/Microsoft.Storage/stable/2023-05-01/storage.json#L661